sdk/python/generative-ai/rag/notebooks/custom_crack_and_chunk_parallel/crack_and_chunk_with_doc_intel/crack_and_chunk_parallel.py

```diff
@@ -0,0 +1,149 @@
+import os
+import pandas as pd
+import numpy as np
+import argparse
+import mlflow
+import traceback
+from pathlib import Path
+import shutil
+
+from azureml.rag.documents.cracking import file_extension_loaders
+from azureml.rag.tasks.crack_and_chunk import (
+    __main__,
```

Comment:
Normally, we shouldn't import any private function ("__main__" in this case), because package/model owner could change private function (even remove private function ) at any time.